### DISCUSSION TITLE
How to change the AIO login password?

### DISCUSSION THREAD

**Discussion by c469591** [nextcloud/all-in-one#3368] [Questions] [answered]
Hello, I have successfully set up according to this instruction.
https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md
But I access it using a browser.
ip:8080
In the AIO page, it is necessary to enter a password.
How should I configure my login password?
thanks

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/discussions/1786
